pmm-app/src/pmm-dbaas/components/DBCluster/DBCluster.utils.tsx

```diff
@@ -12,3 +15,13 @@ export const getClusterStatus = (
 
   return key || DBClusterStatus.failed;
 };
+
+export const buildWarningMessage = (className: string) => (
+  <>
+    {Messages.dbcluster.publicAddressWarningBegin}
+    &nbsp;
+    <a href={ADVANCED_SETTINGS_URL} className={className}>{Messages.dbcluster.publicAddressWarningLink}</a>
+    &nbsp;
```

Comment:
Are these nbsp necessary here? Can't some CSS be used instead?

Reply:
I need the nbsp because I'm breaking the string into multiple lines to don't have a gigantic string but still want to get the spaces. Sure it can me done with CSS, but since the goal of this is to add a link to a middle of a string I would have to try to match with margin/padding the same spacing that you get with nbsp.